4_arm.ipynb

```diff
@@ -644,6 +1149,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
+    "    <p>In this notebook we have teached the robot about the object positions.</p>\n",
+    "    <p>In the next notebook, we will learn how to detect the object form the camera image and write a code to pick the object automatically.</p>\n",
```

Comment:
Here, I am assuming there is just a single object being detected.  **If there are multiple, we must modify this sentence.**

    "    <p>In the next notebook, we will learn how to detect an object using the camera image and write code to pick up the object automatically.</p>\n",

Reply:
In this specific notebook, we only explains image recognizer that can only detect single object at a time. So your understanding is correct.